mapproxy/service/templates/wmts100capabilities.xml

```diff
@@ -97,6 +97,12 @@
       <ows:Identifier>{{layer.name}}</ows:Identifier>
       <Style>
         <ows:Identifier>default</ows:Identifier>
+        {{if layer.legend_version}}
+        <LegendURL
+          format="image/png"
+          xlink:href="{{service.url}}?service=WMS&amp;request=GetLegendGraphic&amp;version={{layer.legend_version}}&amp;format=image%2Fpng&amp;layer={{layer.name}}"
```

Comment:
Can we safely assume that `format=image/png` is supported?

Reply:
Yes, MapProxy automatically converts image formats. We could also advertise all other possible image formats if we want to.

Reply:
I will leave it at png for now, if other formats are needed we can change accordingly